README.md

```diff
@@ -41,6 +41,7 @@ Here is a [step-by-step guide](docs/getting-started-tutorial.md) showing you how
 * **Incremental updates**: Automated replications are based on incremental updates to reduce your data transfer costs.
 * **Manual full refresh**: Sometimes, you need to re-sync all your data to start again.
 * **Debugging autonomy**: Modify and debug pipelines as you see fit, without waiting.
+* **Integration extensibility:** Adapt an existing integration to your needs or build a new one with ease.
```

Comment:
Two suggestions: 
1. replace `Integration extensibility` with `Built for extensibility`
1. make this the first or second bullet point in this list

Reply:
1. Done
2. Put in 2nd point.